Dockerfile

```diff
@@ -1,67 +1,24 @@
-####### TEST BUILD #######
```

Comment:
WHere are you installing sudo here as we discussed?

Reply:
removed here 4f064a18bc03200e75a3aac269586258d520cc7b

But still there are some issues with CI during the container scanning (only with dashing?)